### PR TITLE
nethserver-install: enable nethserver-config-network

### PR DESCRIPTION
**Pull request by gsanchietti**
NethServer/dev#5520

### CODE REVIEW

File: root/usr/sbin/nethserver-install [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+G#!/bin/bash
```
Comment (DavidePrincipi) [inline root/usr/sbin/nethserver-install]: Too big TYPO here :smiley_cat:

File: root/usr/sbin/nethserver-install [hunk 2]
```diff
@@ -108,5 +108,6 @@ for UNIT in NetworkManager firewalld; do
         logexec "systemctl preset $UNIT"
     fi
 done
+systemctl enable nethserver-config-network
```
Comment (DavidePrincipi) [inline root/usr/sbin/nethserver-install]: Should use `logexec` wrapper consistently, like other commands

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]
:-1:

**Comment by gsanchietti** [comment]
It was a very bad patch! Please, check it now :)

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**